include/ews/ews.hpp

```diff
@@ -17072,6 +17146,44 @@ class basic_service final
         return create_item_impl(the_message, disposition, folder);
     }
 
+    //! \brief Creates new messages in the Exchange store.
+    //!
+    //! Creates new messages and, depending of the chosen message
+    //! disposition, sends them to the recipients.
+    //!
+    //! Note that if you pass message_disposition::send_only or
+    //! message_disposition::send_and_save_copy this function always
+    //! returns an invalid item id because Exchange does not include
+    //! the item identifier in the response. A common workaround for this
+    //! would be to create the item with message_disposition::save_only, get
+    //! the item identifier, and then use the send_item to send the message.
+    //!
+    //! \return A vector of the item ids of the saved messages when
+    //! message_disposition::save_only was given; otherwise an invalid item
+    //! id.
+    std::vector<item_id> create_item(const std::vector<message>& messages,
+                                     ews::message_disposition disposition)
+    {
+        return create_item_impl(messages, disposition, folder_id());
+    }
+
+    //! \brief Creates new messages in the specified folder.
+    //!
+    //! \param messages The message items that are about to be created.
+    //! \param disposition Whether the message is only saved, only send, or
+    //! saved and send.
+    //! \param folder The target folder where the message is saved.
+    //!
+    //! \return A vector of the item ids of the saved messages when
+    //! message_disposition::save_only was given; otherwise an invalid item
+    //! id.
+    std::vector<item_id> create_item(const std::vector<message>& messages,
+                                     ews::message_disposition disposition,
+                                     const folder_id& folder)
```

Comment:
Again, do we really return a vector of invalid Item_ids here? Docs are a little unspecific.